src/components/FavoritesSidebar/index.js

```diff
@@ -0,0 +1,27 @@
+import { useContext } from "react";
+import { CloseFavoritesSidebar, FavoriteSidebar, FavoritesLinks, FavoritesTitle, FavoritesUl } from "./styled";
+import { GlobalContext } from "../../context/GlobalContext";
+
+export default function Sidebar({initialLikes, setShowSidebar}) { 
+    const { likes } = useContext(GlobalContext);
```

Comment:
No need to touch GlobalContext here. Firebase can update this list automaticaly.

```
import { getDatabase, ref, onChildAdded, onChildChanged, onChildRemoved } from "firebase/database";

const db = getDatabase();
const commentsRef = ref(db, 'post-comments/' + postId);
onChildAdded(commentsRef, (data) => {
  addCommentElement(postElement, data.key, data.val().text, data.val().author);
});

onChildChanged(commentsRef, (data) => {
  setCommentValues(postElement, data.key, data.val().text, data.val().author);
});

onChildRemoved(commentsRef, (data) => {
  deleteComment(postElement, data.key);
});
```

Please refer to [this documentation](https://firebase.google.com/docs/database/web/lists-of-data?hl=pt&authuser=0#web-version-9_2) for more details.